### DISCUSSION TITLE
Option for copy path for media and rich text editor

### DISCUSSION THREAD

**Discussion by sardarms** [sveltia/sveltia-cms#89] [Ideas]
I have two requests (from a common user perspective): 

1. Please make an option to allow copying the path of the media that is uploaded in the media folder. 
2. Please enable the rich text editor for editing any file. 

These two options are already available on the Decap CMS. 

Thank you in advance.

**Comment by kyoshino** [comment]
Hi @sardarms, thanks for checking out Sveltia CMS!

Copy button: I thought I listed it as a known issue on README but looks like I haven’t. One of my clients still uses Netlify CMS, and I know the Copy Path and Download buttons are readily available — I’ll implement these buttons soonish. 

<img width="428" alt="image" src="https://github.com/sveltia/sveltia-cms/assets/2929505/fa2bf7b0-a5a0-42cb-829b-10fb1054c1dc">

The rich text editor will soon also be added. It’s tracked in https://github.com/sveltia/sveltia-cms/issues/32 🙂

**Comment by kyoshino** [comment]
@sardarms Sorry to keep you waiting! The copy button has been added with [v0.14.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.14.0).
